iree/compiler/Dialect/HAL/Transforms/MaterializeInterfaces2.cpp

```diff
@@ -1192,8 +1193,8 @@ class MaterializeInterfaces2Pass
 };
 
 std::unique_ptr<OperationPass<ModuleOp>> createMaterializeInterfaces2Pass(
-    TargetOptions targetOptions) {
-  return std::make_unique<MaterializeInterfaces2Pass>(targetOptions);  // NOLINT
+    TargetOptions executableOptions) {
+  return std::make_unique<MaterializeInterfaces2Pass>(executableOptions);  // NOLINT
```

Comment:
To make clang-format happy I guess

```suggestion
  return std::make_unique<MaterializeInterfaces2Pass>(
      executableOptions);  // NOLINT
```

I'm also going to ask why this is marked `NOLINT` at all.